File: Jour16/jour16.c
#include<stdio.h>

int main()
{
	int Tab[100];
	int i,n;
	do
	{
		printf("Donner le nombre d'entier a saisir : ");
		scanf("%d",&n);
		if(n<0 || n>100)
			printf("Saisie invalide n compris entre 0 et 100\n");
	}while(n<0 || n>100);
	
	for(i = 0; i < n; i++)
	{
		printf("Donner l'element %d : ",i+1);
		scanf("%d",Tab+i);
	}

	printf("VOICI VOS DONNEES SAISIES\n\n");
	for(i = 0; i < n; i++)
	{
		printf("[%d]",Tab[i]);
		
	}
	return 0;
}

File: Jour16/jour16.py
#coding : utf-8
Liste = []

while 1:
	n = input("Donner le nombres d'entier a saisir :")
	try:
		n = int(n)
		if n < 0 or n > 100:
			print("Erreur :::: Donner un entier positf.")
			continue
	except:
		print("Saisie invalide!!!")
		continue
	else:
		break

for i in range(n):
	while 1:
		pousseur = input(f"Donner l'element {i+1} :")
		try:
			pousseur = int(pousseur)
		except:
			print("Saisie invalide!!!")
			continue
		else:
			break

	Liste.append(pousseur)	

print("\nVOICI VOS DONNEES SAISIES\n")
for i in range(n):
	print(Liste[i])